src/assets/github.tsx

```diff
@@ -0,0 +1,20 @@
+import { rem } from "@mantine/core";
+
+interface IconProps extends React.ComponentPropsWithoutRef<"svg"> {
+  size?: number | string;
+}
+
+export const GithubIcon = ({ size, style, ...others }: IconProps) => {
```

Comment:
```suggestion
export const GithubIcon: FC<IconProps> = ({ size, style, ...others }) => {
```

давай так писать, чтобы еще return значение компонента типизировалось